config/config.go

```diff
@@ -20,13 +20,14 @@ import (
 
 // Install contains the install time configuration of the server and kubernetes dependency
 type Install struct {
-	config.Install `yaml:",inline"`
-	config.Runtime `yaml:",inline"`
-	Kubeconfig     string  `yaml:"kube-config,omitempty"`
-	FIFO           bool    `yaml:"fifo,omitempty"`
-	QPS            float32 `yaml:"qps,omitempty"`
-	Burst          int     `yaml:"burst,omitempty"`
-	BinpackAlgo    string  `yaml:"binpack,omitempty"`
+	config.Install     `yaml:",inline"`
+	config.Runtime     `yaml:",inline"`
+	Kubeconfig         string  `yaml:"kube-config,omitempty"`
+	FIFO               bool    `yaml:"fifo,omitempty"`
+	QPS                float32 `yaml:"qps,omitempty"`
+	Burst              int     `yaml:"burst,omitempty"`
+	BinpackAlgo        string  `yaml:"binpack,omitempty"`
+	InstanceGroupLabel string  `yaml:"InstanceGroupLabel,omitempty"`
```

Comment:
Two things here:
1. to avoid a breaking change, we should set this to a default `resource_channel` value
2. (nit) we should have this key in the same case as the rest of the config rather than Pascal Case, for consistency